string blind
int16 val
---
int16 lift